src/main/java/vg/civcraft/mc/civmodcore/dao/ManagedDatasource.java

```diff
@@ -62,7 +66,8 @@
 	private static final long MAX_WAIT_FOR_LOCK = 600000l;
 	private static final long WAIT_PERIOD = 500l;
 	
-	private ConnectionPool connections;
+	private HikariDataSource connections;
+	private HikariConfig config;
```

Comment:
I don't think the HikariConfig is needed after the datasource has been created. I'm not sure this field is even used.